eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Serialisation/Tripping/CBOR.hs

```diff
@@ -273,7 +273,7 @@ tests =
       testProperty "roundtrip Epoch State" prop_roundtrip_EpochState,
       testProperty "roundtrip NewEpoch State" prop_roundtrip_NewEpochState,
       testProperty "roundtrip MultiSig" prop_roundtrip_MultiSig,
-      testProperty "roundtrip Metadata" prop_roundtrip_metadata,
+      testProperty "roundtrip Metadata" prop_roundtrip_auxData,
```

Comment:
Should we also change the description to `roundtrip AuxData` or it's been left for a reason ?

Reply:
I think we should change it too :)

Reply:
No you are right. I changed it.